joern-cli/src/main/scala/io/shiftleft/joern/JoernParse.scala

```diff
@@ -85,12 +111,13 @@ object JoernParse extends App {
                           enhance: Boolean = true,
                           dataFlow: Boolean = true,
                           enhanceOnly: Boolean = false,
-                          language: String = "c")
+                          language: String = "",
+                          listLanguages: Boolean = false)
 
   def parseConfig(parserArgs: List[String]): Either[String, ParserConfig] =
     new scopt.OptionParser[ParserConfig]("joern-parse") {
       arg[String]("input")
-        .required()
+        .optional()
```

Comment:
why should this be optional?

Reply:
It should be possible to run `joern-parse --list-languages` without providing an input path. If the `--list-languages` option isn't specified, there's a check later to make sure that a valid input path has been provided.

Reply:
ah, ok, makes sense :+1: